src/ServiceControlInstaller.PowerShell/ServiceControlInstaller.Powershell.dll-help.xml

```diff
@@ -833,77 +921,78 @@ Get-SecurityIdentifier MyDomain\Foo</dev:code>
+					<maml:para>The directory that will contain the RavenDB database for this ServiceControl Instance, This should be unique for each instance of ServiceControl and should not be a subfolder of the InstallPath or LogPath</maml:para>
 				</maml:description>
 				<command:parameterValue required="true" variableLength="false">String</command:parameterValue>
 			</command:parameter>
 			<command:parameter required="true" variableLength="false" globbing="false" pipelineInput="false" position="named">
 				<maml:name>LogPath</maml:name>
 				<maml:description>
-					<maml:para>The directory to use for this ServiceControl Logs</maml:para>
+					<maml:para>The directory that will contain the logs for this instance of ServiceControl.  This should be unique for each instance of ServiceControl and should not be a subfolder of the InstallPath or DBPath</maml:para>
 				</maml:description>
 				<command:parameterValue required="true" variableLength="false">String</command:parameterValue>
 			</command:parameter>
 			<command:parameter required="false" variableLength="false" globbing="false" pipelineInput="false" position="named">
 				<maml:name>HostName</maml:name>
 				<maml:description>
-					<maml:para>The hostname to use in the URLACL (defaults to localhost)</maml:para>
+					<maml:para>The hostname to use in the URLACL (defaults to localhost).  Anyone who can access the ServiceControl URL has complete access to the message data stored within ServiceControl. This is why the default is to only respond to the localhost. Consider carefully the implications of exposing ServiceControl via a custom or wildcard URI.</maml:para>
 				</maml:description>
 				<command:parameterValue required="true" variableLength="false">String</command:parameterValue>
 			</command:parameter>
 			<command:parameter required="true" variableLength="false" globbing="false" pipelineInput="false" position="named">
 				<maml:name>Port</maml:name>
 				<maml:description>
-					<maml:para>The port number to listen on. If this is the only ServiceControl instance then 33333 is recommended</maml:para>
+					<maml:para>The TCP port number to listen on. If this is the only ServiceControl instance then 33333 is recommended.  The port number must be unique for each instance and not be in use by another process.  Use the Test-IfPortIsAvailable cmdlet to verify a port number is free</maml:para>
 				</maml:description>
 				<command:parameterValue required="true" variableLength="false">Int32</command:parameterValue>
 			</command:parameter>
 			<command:parameter required="false" variableLength="false" globbing="false" pipelineInput="false" position="named">
 				<maml:name>ErrorQueue</maml:name>
 				<maml:description>
-					<maml:para>The ErrorQueue name to consume messages from. Default is error</maml:para>
+					<maml:para>Specifies the input queue name to consume error messages from.</maml:para>
```

Comment:
Intentional  - There is a tag specifically to contain the default value so no need to have it the description